app/src/main/java/com/realityexpander/tasky/MainActivity.kt

```diff
@@ -42,63 +44,83 @@ class MainActivity : ComponentActivity() {
     private val viewModel: MainActivityViewModel by viewModels()
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        if(false) {
+        if (false) {
```

Comment:
I assume just testing? Then this shouldn't make its way into a PR 😄

Reply:
Yes just for testing and I was trying to limit the size of the PR, and left this in as a work-in-progress, otherwise the PR will be even bigger!

Otherwise, I will have to remove this, make the PR, then put it back in to keep going.

What is the correct way to do keep items that I am still working on active and keep the PR small?

Reply:
Well ideally in a real work environment the PRs should be small enough to not need such test code (this one was still ~5x as big as the suggested max size). Here in this project it's not an issue since only you are working on it, but imagine you'd have coworkers and want to merge code like `if(false)` into the production branch. Such code has nothing lost there 😄

Reply:
OK, yes, i would not do that while on a team. Still working on making the PR's smaller.